src/textual/app.py

```diff
@@ -641,6 +641,7 @@ async def press_keys() -> None:
                             await asyncio.sleep(0.02)
 
                     await app._animator.wait_for_idle()
+                    await asyncio.sleep(0.05)
```

Comment:
On Windows, screenshots seemed to be slightly too early and the final key wasn't being processed. Not ideal of course, but seems to fix the issue.

Reply:
THis might be redundant with a pending PR.